### DISCUSSION TITLE
Bugs in V2.8.0

### DISCUSSION THREAD

**Discussion by Josephred999** [argoproj/argo-cd#15073] [General]
After upgrading our ArgoCD Instances to v2.8.0, I cant see my applicationset list

`PermissionDenied desc = error listing ApplicationSets with selectors: applicationsets.argoproj.io is
 forbidden: User \"system:serviceaccount:argocd:argocd-server\" cannot list resource \"applicationsets\" in API group \"argoproj.io\" at the cluster scope"`

**Comment by crenshaw-dev** [comment]
Reported here. Thanks! https://github.com/argoproj/argo-cd/issues/14903

**Reply by Josephred999** [reply]
Aaah ok ok. So, it's a known bug and the fix in en-route.
Appreciate the reply, Michael.
